Doc/whatsnew/3.8.rst

```diff
@@ -595,6 +595,30 @@ The :func:`ast.parse` function has some new flags:
 asyncio
 -------
 
+:func:`asyncio.run` has graduated from the provisional to stable API. This
+function can be used to execute a :term:`coroutine` and return the result while
+automatically managing the event loop. For example:
+
+    >>> async def main():
+    ...     await asyncio.sleep(0)
+    ...     return 42
+    >>> asyncio.run(main())
+    42
+
+This is *roughly* equivalent to:
```

Comment:
I'd also add something along the "the actual implementation is more complex and therefore asyncio.run should be the preferred way to run asyncio programs"